envi/archs/ppc/__init__.py

```diff
@@ -66,7 +66,7 @@ def _getLSBSet(value):
     for i, c in zip(range(len(binvalstr)), reversed(binvalstr)):
         if c == '1':
             return i
-    return None
+    return 0
```

Comment:
honestly, i'm not sure why this is the way that it is, but it's certainly not a fast implementation.

better to keep the thing *all* math.  strings in python are inherently slow.

Reply:
also, what about value == 0?  what's the proper response?  being 0 seems, odd.
possibly:

```
def _getLSBSet(value):
    '''
    Determine the least significant bit of a value
    If value is zero, there is no least significant bit
    '''
    lsb = 0
    while not (value & 1) and value != 0:
        value >>= 1
        lsb += 1
    return lsb
```

this returns 0 for the (value == 0) case, so the code results should be identical, but this should be much faster.

does it make sense to return -1  for the (value==0) case?